src/TuringGLM.jl

```diff
@@ -15,17 +17,35 @@ using StatsModels: StatsModels
 using Tables: Tables
 using TableOperations: TableOperations
 
+# LazyArrays
+using LazyArrays: @~, LazyArray
```

Comment:
Did this help performance? Otherwise maybe keep things simple

Reply:
Yes, it does.